CHANGELOG.md

```diff
@@ -4,6 +4,57 @@ All notable changes to this project will be documented in this file.
 
 Breaking changes marked with a :boom:
 
+## [1.6.0] - 2023-01-30
+
+### Features
+
+- [`workflow`] (Experimental) Introduced a major optimization to the workflow runtime ([#951](https://github.com/temporalio/sdk-typescript/pull/951)).
```

Comment:
@bergundy Anything else you would like to mention here?

Reply:
This is good, thanks